pkg/vm/pipeline/types.go

```diff
@@ -79,8 +83,22 @@ import (
 //  -> π(c.city, s.city, revenue)
 type Pipeline struct {
 	// attrs, column list.
-	attrs []string
+	attrs         []string
```

Comment:
Put it in an independent struct.
`
        attrs         []string
	cols          []*plan.ColDef
	Name2ColIndex map[string]int32
	CreateSql     string
`